include/tracelogging/TraceLoggingProvider.h

```diff
@@ -273,12 +256,12 @@ A symbol declared by TRACELOGGING_DECLARE_PROVIDER must later be defined in a
 .c or .cpp file using the TRACELOGGING_DEFINE_PROVIDER macro.
 */
 #define TRACELOGGING_DECLARE_PROVIDER(providerSymbol)                                                                                                                 \
-    _tlg_EXTERN_C struct lttng_ust_tracepoint *__start___tracepoints_ptrs_##providerSymbol[] __attribute__((weak, visibility("hidden")));                             \
-    _tlg_EXTERN_C struct lttng_ust_tracepoint *__stop___tracepoints_ptrs_##providerSymbol[] __attribute__((weak, visibility("hidden")));                              \
-    _tlg_EXTERN_C struct lttng_event_desc const *__start___eventdesc_ptrs_##providerSymbol[] __attribute__((weak, visibility("hidden")));                             \
-    _tlg_EXTERN_C struct lttng_event_desc const *__stop___eventdesc_ptrs_##providerSymbol[] __attribute__((weak, visibility("hidden")));                              \
+    _tlg_EXTERN_C struct lttng_ust_tracepoint *_tlg_PASTE(__start___tracepoints_ptrs_, providerSymbol)[] __attribute__((weak, visibility("hidden")));                             \
```

Comment:
Using `_tlg_PASTE` instead of `##` is not strictly part of the 2.13 work. It enables code like the following:

```C
#define MyProvider MyProviderName
TraceLoggingWrite(MyProvider, ...);
```